### PR TITLE
Add description of verify for modbus switch.

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Describe the change to "verify:" in "switch:".

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/50117
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -710,7 +710,7 @@ switches:
       type: integer
       default: 0
     verify:
-      description: read from modbus device to verify switch.
+      description: read from modbus device to verify switch. If used without attributes it used the toogle register configuration. If omitted no verify is done, but the state of the switch is set with each toogle.
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
      description: read from modbus device to verify switch. If used without attributes it used the toggle register configuration. If omitted no verify is done, but the state of the switch is set with each toggle.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (janiversen) [reply source/_integrations/modbus.markdown]: Thanks for correcting my english 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by janiversen** [comment]
@frenck am I correct in assuming “next” is always the next release for now 2021.6 and never patch releases ?
If I am correct how is the it managed in the time where we prepare a new release...eg. I submit a PR for next 1/5 but it is for 2021.6 and not 2021.5.

I apologize for being curious, but the more I understand the better I can help.

**Comment by janiversen** [comment]
Ups pressed the wrong button.

File: source/_integrations/modbus.markdown [hunk 2]
```diff
@@ -710,7 +710,7 @@ switches:
       type: integer
       default: 0
     verify:
-      description: read from modbus device to verify switch.
+      description: read from modbus device to verify switch. If used without attributes it used the toggle register configuration. If omitted no verify is done, but the state of the switch is set with each toggle.
```
Comment (MartinHjelmare) [inline source/_integrations/modbus.markdown]:
```suggestion
      description: Read from modbus device to verify switch. If used without attributes it uses the toggle register configuration. If omitted no verification is done, but the state of the switch is set with each toggle.
```

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
> @frenck am I correct in assuming “next” is always the next release for now 2021.6 and never patch releases ?

`next` = for the next major release (2021.6 at the time of writing).
`current` = for the current release (2021.5.x at the time of writing).

**Pull request merged**